server.c

```diff
@@ -1562,6 +1562,14 @@ void fio_server_send_ts(struct thread_stat *ts, struct group_run_stats *rs)
 	p.ts.ss_deviation.u.i	= cpu_to_le64(fio_double_to_uint64(ts->ss_deviation.u.f));
 	p.ts.ss_criterion.u.i	= cpu_to_le64(fio_double_to_uint64(ts->ss_criterion.u.f));
 
+	p.ts.priority_bit = ts->priority_bit;
+	for (i = 0; i < FIO_IO_U_PLAT_NR; i++) {
+		p.ts.io_u_plat_prio[i] = cpu_to_le64(ts->io_u_plat_prio[i]);
```

Comment:
Isn't it technically high priority and regular priority rather than low priority and regular priority?

Reply:
Yes, you are correct. I'll rename the variables and try not to confuse the division between the functional and the logging commits.